tests/performance-tests/src/tests/public/permalinkPage.test.ts

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable no-console */
+import { check, fail } from 'k6';
+import http, { RefinedResponse, ResponseType } from 'k6/http';
+import { Counter, Rate, Trend } from 'k6/metrics';
+import { Options } from 'k6/options';
+import getEnvironmentAndUsersFromFile from '../../utils/environmentAndUsers';
+import loggingUtils from '../../utils/loggingUtils';
+
+// slowest permalink on dev at the time of writing
+// (after changing max table size to 6 million)
+// created using absence by geographic level data
+const permalinkId = '010a5b4a-1d1a-4379-4abf-08db14bee00d';
+const permalinkTitle = "'1' from 'big permalink'";
+
+export const options: Options = {
+  stages: [
+    {
+      duration: '0.1s',
+      target: 50,
+    },
+    {
+      duration: '10m',
+      target: 40,
+    },
+  ],
+  noConnectionReuse: true,
+  insecureSkipTLSVerify: true,
+};
+
+export const errorRate = new Rate('ees_errors');
+export const getPermalinkSuccessCount = new Counter(
+  'ees_get_permalinkpage_success',
+);
+export const getPermalinkFailureCount = new Counter(
+  'ees_get_permalinkpage_failure',
+);
+export const getPermalinkPageRequestDuration = new Trend(
+  'ees_get_permalinkpage_duration',
+  true,
+);
+
+const environmentAndUsers = getEnvironmentAndUsersFromFile(
```

Comment:
It seems a shame we have to run "npm run store-environment-details" just so we can get the environment details here - I might see if there's any chance to address this in the Public API mock perf test writing.